tests/mock_network/requests.rs

```diff
@@ -227,77 +225,3 @@ fn failed_get_request() {
 
     assert_eq!(response_received_count, 1);
 }
-
-#[test]
-// TODO (quic-p2p): this test relies on a behaviour of mock-crust which drops all in-flight messages
-// on disconnect. Verify whether this behaviour should be transitioned over to quic-p2p. If yes,
-// then do so and then re-enable this test. If not, modify this test accordignly (or remove it if
-// not relevant anymore).
-#[ignore]
-fn disconnect_on_get_request() {
```

Comment:
the title of the PR indicates this ignored test shall be moved to somewhere else.
however it seems not appear among the above new created tests?

Reply:
If you look at it commit by commit, some are moved and some are removed. When tests are removed, Adam explained why in the appropriate commit messages.